subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -1213,6 +1213,30 @@ int lwm2m_rd_client_connection_resume(struct lwm2m_ctx *client_ctx)
 }
 #endif
 
+int lwm2m_rd_client_server_not_respond(struct lwm2m_ctx *client_ctx)
```

Comment:
```suggestion
int lwm2m_rd_client_timeout(struct lwm2m_ctx *client_ctx)
```

I think we should name the function to show what it does, not when it is used.